File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/TableModel.java
package kz.greetgo.sandbox.controller.model;

public class TableModel {
    public int id;
    public String fullName;
    public String charm;
    public long age;
    public double totalBalance;
    public double minBalance;
    public double maxBalance;

    @Override
    public String toString(){
        return "\n\t\t\t"+id +" "+ fullName +" "+ charm +" "+ age +" "+ totalBalance +" "+ minBalance +" "+ maxBalance+"\n";
    }




}

File: sandbox.client/src/models/TableModel.ts
import {CharmType} from "./CharmType";

export class TableModel{
    public id:string;
    public fullName:string;
    public charm:CharmType|null;
    public age:number;
    public totalBalance:number;
    public maxBalance:number;
    public minBalance:number;

    public assign(o: any): TableModel{
      this.id = o.id;
      this.fullName = o.fullName;
      this.charm = o.charm;
      this.age = o.age;
      this.totalBalance = o.totalBalance;
      this.maxBalance = o.maxBalance;
      this.minBalance = o.minBalance;
      return this;
    }

    public static copy(a: any): TableModel{
      let ret = new TableModel();
      ret.assign(a);
      return ret;
    }
}
